File: exercises/beecrowd/1173/1173.ts
// https://judge.beecrowd.com/pt/problems/view/1173
import * as fs from "fs";

function main(): void {
  const input = fs.readFileSync("/dev/stdin", "utf8").trim();
  let v = parseInt(input);

  for (let i = 0; i < 10; i++) {
    console.log(`N[${i}] = ${v}`);
    v *= 2;
  }
}

main();

File: exercises/beecrowd/1173/1173.go
// https://judge.beecrowd.com/pt/problems/view/1173
package main

import "fmt"

func main() {
	var v int
	fmt.Scan(&v)

	for i := 0; i < 10; i++ {
		fmt.Printf("N[%d] = %d\n", i, v)
		v *= 2
	}
}
